app/views/items/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="row text-center">
+  <h1><%= @item.title %></h1>
+  <%= image_tag(@item.image) %>
```

Comment:
nice work here @MikelSage ..... think If we put a css class on this image tag and found a way to overlay the image with somekind of retired image it would work?? I have a feeling its not as easy as it sounds sadly. lol